lib/install/tailwindcss.rb

```diff
@@ -38,19 +38,8 @@
   copy_file "#{__dir__}/application.tailwind.css", "app/assets/stylesheets/application.tailwind.css"
 end
 
-if Rails.root.join("Procfile.dev").exist?
-  append_to_file "Procfile.dev", "css: bin/rails tailwindcss:watch\n"
-else
-  say "Add default Procfile.dev"
-  copy_file "#{__dir__}/Procfile.dev", "Procfile.dev"
-
-  say "Ensure foreman is installed"
-  run "gem install foreman"
-end
-
-say "Add bin/dev to start foreman"
-copy_file "#{__dir__}/dev", "bin/dev"
-chmod "bin/dev", 0755, verbose: false
+say "Add puma plugin"
+append_to_file "config/puma.rb", %(plugin :tailwindcss if Rails.env.development?)
```

Comment:
Kind of nitpicky, but what do you think about adding a blank line before and after the plugin? And a brief comment saying what the plugin does. For example, Rails default has this plugin like this:

```ruby
# Allow puma to be restarted by `bin/rails restart` command.
plugin :tmp_restart
```